test/ScanForErrorMiddleware/ErrorMiddlewareFilterTest.php

```diff
@@ -10,6 +10,7 @@
 use ArrayIterator;
 use DirectoryIterator;
 use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
```

Comment:
IDEs can get suggestions when you use a FQCN as well:

```php
/** @var \org\bovigo\vfs\vfsStreamDirectory */
```

I'd argue we could do the same with `ProphecyInterface`. Since they're only being used in annotations for typehints, I'd prefer that.